naga/tests/out/msl/boids.msl

```diff
@@ -56,7 +56,7 @@ kernel void main_(
     metal::float2 _e14 = particlesSrc.particles[index].vel;
     vVel = _e14;
     bool loop_init = true;
-    while(true) {
+    while(true) {__asm__("");
```

Comment:
nitpick(style): Do we not want this indented in-line with the rest of the `while` loop's contents?

Won't block either way, but I didn't see anyone call this out yet.